### PR TITLE
[bitnami/dataplatform-bp1] Chart standardised

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/dataplatform-bp1 chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/dataplatform-bp1/values.yaml [hunk 1]
```diff
@@ -27,6 +39,24 @@ commonAnnotations: {}
 ## @param extraDeploy Array of extra objects to deploy with the release
 ##
 extraDeploy: []
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+
+## Enable diagnostic mode in the deployment(s)/statefulset(s)
+##
+diagnosticMode:
```
Comment (fmulero) [inline bitnami/dataplatform-bp1/values.yaml:48]: Could you add some information in NOTES.txt about the `diagnosticMode`?

**Review by fmulero — CHANGES_REQUESTED** [review]
Thanks @migruiz4 only a little note about `diagnosticMode`

File: bitnami/dataplatform-bp1/values.yaml (hunk 1 above)
Comment (migruiz4) [reply bitnami/dataplatform-bp1/values.yaml]: I had to drop diagnosticMode because dataplatform-bp1 uses `scratch` containers

**Comment by carrodher** [comment]
Unfortunately, we are deprecating both Blueprints Helm charts, so this PR is not needed anymore.
Thanks for your time here 😢

**Pull request closed without merging**